web/packages/build/package.json

```diff
@@ -67,7 +67,7 @@
     "jest-styled-components": "^7.2.0",
     "jsdom": "^21.1.0",
     "jsdom-testing-mocks": "^1.13.0",
-    "msw": "^0.47.4",
+    "msw": "^1.3.3",
```

Comment:
msw 1.3.3 breaks mocks in stories unfortunately. Take a look at http://localhost:9002/?path=/story/teleport-discover-connectmycomputer-setupconnect--polling-success

https://github.com/gravitational/teleport/blob/629f2d347f35d7ca26d69a7eac14dc47e650b4b8/web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.story.tsx#L99-L116

After the update, I'm getting a warning about an outdated service worker telling me to run `msw init`. This is a red herring, it should go away after a hard reset or just by opening the storybook in an incognito window.

After getting through that, I see the following warning:

```
[MSW] Warning: captured a request without a matching request handler:

  • GET http://localhost:9002/v1/webapi/sites/aws/nodes

If you still wish to intercept this unhandled request, please create a request handler for it.
Read more: https://mswjs.io/docs/getting-started/mocks 6 main.iframe.bundle.js:173362:11


XHR GET http://localhost:9002/v1/webapi/sites/aws/nodes?searchAsRoles=&limit=50&startKey=&query=labels["teleport.dev/connect-my-computer/owner"] == "llama"&search=&sort=
[HTTP/1.1 404 Not Found 1ms]
```

Before that, I see another one related to what I assume is the actual problem:

> [MSW] Found a redundant usage of query parameters in the request handler URL for "GET /v1/webapi/sites/:clusterId/nodes?searchAsRoles=:searchAsRoles?&limit=:limit?&startKey=:startKey?&query=:query?&search=:search?&sort=:sort?". Please match against a path instead and access query parameters in the response resolver function using "req.url.searchParams".